tests/projects/test_mixins.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+from django.views.generic import ListView
+from tests.apps.blog import models
```

Comment:
I would suggest to use a relative import. `apps.blog` is also part of the test suite.